Docs: Update doc for PostgreSQL authentication (#31434)
grafana/grafana: docs/sources/administration/provisioning.md

| tsdbVersion             | string  | OpenTSDB                                                         | Version                                                                                     |
| tsdbResolution          | string  | OpenTSDB                                                         | Resolution                                                                                  |
| sslmode                 | string  | PostgreSQL                                                       | SSLmode. 'disable', 'require', 'verify-ca' or 'verify-full'                                 |

> | tlsConfigurationMethod  | string  | PostgreSQL                                        | How SSL Certificate is going to be configured 'file-path' or 'file-content'                       |

Reviewer: ```suggestion
| tlsConfigurationMethod  | string  | PostgreSQL                                        | SSL Certificate configuration, either by 'file-path' or 'file-content'                       |
```
